src/widgets/main/proof/style.module.scss

```
@@ -0,0 +1,157 @@
+	}
+}
+
+@include token.expanded {
+	.proof {
+		grid-template-columns: 1.1fr 0.9fr;
+	}
+}
+
+.proof_copy {
+	@include token.flex_column;
+	gap: token.$spacing_20;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)